libeventheader-decode-cpp/src/EventFormatter.cpp

```diff
@@ -314,23 +507,22 @@ class StringBuilder
     }
 
     // Requires: there is room for 15 chars.
+    // Reads 4 bytes.
     // Writes 7..15 chars, e.g. [0.0.0.0] or [255.255.255.255].
     void
-    WriteIPv4(uint32_t val) noexcept
+    WriteIPv4(void const* val) noexcept
```

Comment:
Note: Accepting a `uint32_t` makes byte-ordering more complicated than it needs to be at the call site. Make the byte ordering be the concern of this function instead.